tests/e2e/default/telemetry/telemetry.wdio-spec.js

```diff
@@ -50,9 +50,13 @@ describe('Telemetry', () => {
   });
   let reportDocs;
 
-  const DATE_FORMAT = 'YYYY-MM-DD';
   const TELEMETRY_PREFIX = 'telemetry';
-  const todayDBName = `${TELEMETRY_PREFIX}-${moment().format(DATE_FORMAT)}-${user.username}`;
+  const formatDbName = (date) => {
+    const formattedDate = `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`;
```

Comment:
I believe we can just use `moment().format('YYYY-MM-D')` and achieve the same effect. And it makes this more readable and really obvious!

Reply:
I agree, that's what I went for initially but I settled for copy/pasting the date formatting logic in [telemetry.service.ts](https://github.com/medic/cht-core/blob/master/webapp/src/ts/services/telemetry.service.ts#L213). It's a simple change to make though, I'll get the CI running!